simplex/src/main/java/io/em2m/simplex/std/dates.kt

```diff
@@ -0,0 +1,107 @@
+package io.em2m.simplex.std
+
+import io.em2m.simplex.model.*
+import io.em2m.simplex.parser.DateMathParser
+import io.em2m.utils.coerce
+import io.em2m.utils.coerceNonNull
+import org.joda.time.format.DateTimeFormat
+import java.util.*
+
+class FormatDatePipe : PipeTransform {
+
+    private var dateFormat: String = ""
+
+    override fun args(args: List<String>) {
+        if (args.isNotEmpty()) {
+            dateFormat = args[0]
+        }
+    }
+
+    override fun transform(value: Any?, context: ExprContext): Any? {
+        val dateInput : Date? = value?.coerce()
+         return if (dateInput != null) {
+             val pattern = DateTimeFormat.forPattern(dateFormat)
```

Comment:
The pattern should be parsed in the args function instead of here since that will be cached and therefore more efficient.